docs/user-guide/networkpolicies.md

```diff
@@ -64,6 +64,9 @@ spec:
     role: db
  ingress:
   - from:
+     - namespaceSelector:
```

Comment:
I think this is actually OK, since they're in separate `NetworkPolicyPeer`s.

This policy says "allow from any pods in namespaces=project:myproject OR pods=role:frontend"